pytripgui/view/field.ui

```diff
@@ -332,81 +349,88 @@
-        <property name="minimumSize">
+        <property name="maximumSize">
          <size>
-          <width>82</width>
-          <height>0</height>
+          <width>1000</width>
+          <height>16777215</height>
          </size>
         </property>
-        <property name="text">
-         <string>- 90°</string>
+        <property name="value">
+         <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
-      <item row="7" column="1">
-       <widget class="QDoubleSpinBox" name="contourext_doubleSpinBox">
+      <item row="3" column="1">
+       <widget class="QDoubleSpinBox" name="gantry_doubleSpinBox">
         <property name="sizePolicy">
-         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
+        <property name="minimumSize">
+         <size>
+          <width>82</width>
+          <height>0</height>
+         </size>
+        </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>16777215</height>
          </size>
         </property>
+        <property name="minimum">
+         <double>-400.000000000000000</double>
+        </property>
+        <property name="maximum">
+         <double>400.000000000000000</double>
+        </property>
+        <property name="singleStep">
+         <double>1.000000000000000</double>
+        </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
-      <item row="5" column="0">
-       <widget class="QLabel" name="rasterSpace_label">
-        <property name="text">
-         <string>Raster spacing X, Y [mm]</string>
+      <item row="1" column="3">
+       <widget class="QDoubleSpinBox" name="isocenterZ_doubleSpinBox">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
         </property>
-       </widget>
-      </item>
-      <item row="2" column="2">
-       <widget class="QPushButton" name="gantry_pushButton_p90">
         <property name="minimumSize">
          <size>
-          <width>0</width>
+          <width>82</width>
           <height>0</height>
          </size>
         </property>
+       </widget>
+      </item>
+      <item row="2" column="0">
+       <widget class="QLabel" name="anglesStandard_label">
         <property name="text">
-         <string>+ 90°</string>
+         <string>Angles standard</string>
         </property>
        </widget>
       </item>
-      <item row="4" column="0">
-       <widget class="QLabel" name="spotSize_label">
+      <item row="2" column="1">
+       <widget class="QRadioButton" name="anglesStandardTRiP_radio">
```

Comment:
Can you group the radiobuttons using this class: https://doc.qt.io/qt-5/qbuttongroup.html 
This would ensure that only one radio button is toogled
(see i.e. https://www.delftstack.com/tutorial/pyqt5/pyqt5-radiobutton/)

Moreover button group provides a method `checkedButton` which tells you which one is selected